resource_management_mobile/src/app/shared/components/hiring-change-card/hiring-change-card.component.ts

```diff
@@ -0,0 +1,61 @@
+import { CommonModule } from '@angular/common';
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { FormControl, FormGroup, ReactiveFormsModule, Validators } from '@angular/forms';
+import { IonicModule, ModalController } from '@ionic/angular';
+import { StaticDataConstants } from 'src/app/core/constant/staticData.constants';
+import { CommonService } from '../../services/common.service';
+
+@Component({
+  selector: 'app-hiring-change-card',
+  templateUrl: './hiring-change-card.component.html',
+  styleUrls: ['./hiring-change-card.component.scss'],
+  standalone: true,
+  imports: [IonicModule, CommonModule, ReactiveFormsModule]
+})
+export class HiringChangeCardComponent  implements OnInit {
+  @Input() hiringData: any;
+
+  resourceList:any=[];
```

Comment:
Take your time and replace any with relevant type

Reply:
Almost all changed. Except for some listeners.